src/v0/destinations/customerio/util.js

```diff
@@ -295,6 +297,18 @@ const defaultResponseBuilder = (message, evName, userId, evType, destination, me
   return { rawPayload, endpoint, requestConfig };
 };
 
+const validateConfigFields = destination => {
```

Comment:
add function description

Reply:
No description is needed as the function name in itself is enough